src/test/objectstore/test_bluefs.cc

```diff
@@ -435,13 +435,13 @@ TEST(BlueFS, test_simple_compaction_async) {
     for (int i=0; i<10; i+=2) {
        string dir = "dir.";
        dir.append(to_string(i));
-       for (int j=0; j<10; j+=2) {
+       for (int j=0; j<10; j++) {
```

Comment:
Don't remember the reason behind not removing all the files, may be want to fail the rmdir? If we are adding an assert to remove all, please remove the comment above.